### PR TITLE
Add Community openSUSE repository

### PR DESCRIPTION
**Pull request by cabelo**

### CODE REVIEW

**Comment by StefanBruens** [comment]
From a distribution standpoint, I am strictly against this:

- oneVPL is already included in the distribution, package `libVPL`. No need for duplicating it
- any packages not part of the distribution should be submitted to the distribution. "Community" repos tend to fall apart after one or two years (and half of the packages in the repo mentioned are already broken in some way).

@cabelo - Please cooperate with the distribution maintainers to get things integrated properly.

**Comment by StefanBruens** [comment]
@sndirsch

**Comment by cabelo** [comment]
> From a distribution standpoint, I am strictly against this:
> 
>     * oneVPL is already included in the distribution, package `libVPL`. No need for duplicating it
> 
>     * any packages not part of the distribution should be submitted to the distribution. "Community" repos tend to fall apart after one or two years (and half of the packages in the repo mentioned are already broken in some way).
> 
> 
> @cabelo - Please cooperate with the distribution maintainers to get things integrated properly.

@StefanBruens Thank you for the explanation. But...

1 - There is no onevpl for openSUSE.
2 - I am an official openSUSE contributor of the realsense library and others since 2006.
3- I have been the maintainer of all openSUSE images on Amazon since 2017.

(if necessary, send evidence of the above statements)

To become official there are procedures that are in progress. As an Intel Innovator and openSUSE Ambassador, I am doing a free job of packaging onevpl for openSUSE as it is not supported by Intel.

While it does not enter the official repository, I have made the submission repository available.

**Comment by StefanBruens** [comment]
> @StefanBruens Thank you for the explanation. But...
> 
> 1 - There is no onevpl for openSUSE.

Here: https://build.opensuse.org/package/show/openSUSE:Factory/libvpl  
If something is missing, please branch the package, fix it, and then send an SR to update it.

Several other parts are also officially available, see https://build.opensuse.org/project/show/X11:XOrg

> 2 - I am an official openSUSE contributor of the realsense library and others since 2006.

I see you have updated the package twice. But the hard part, i.e. fixing packaging errors, has been done by others.

Contributions are very welcome, but these should go to the distribution proper, not added as a third party repository.

**Comment by cabelo** [comment]
@StefanBruens Sorry for my failure.

**Comment by cabelo** [comment]
> ere: https://build.opensuse.org/package/show/openSUSE:Factory/libvpl
@StefanBruens : 
Can I add this official url in the README file of this request?

**Comment by cabelo** [comment]
> > ere: https://build.opensuse.org/package/show/openSUSE:Factory/libvpl
> > @StefanBruens :
> > Can I add this official url in the README file of this request?
@StefanBruens :

After I update this onevpl repository... 
Can I add this official url in the README file of this request?

**Comment by cabelo** [comment]
@StefanBruens  Fixed the URL, again sorry for my mistake and confusion.

**Comment by StefanBruens** [comment]
Factory is the default repository (actually, its the staging area, i.e. a few days ahead) for Tumbleweed, so Tumbleweed users already have it. (And e.g blender already uses  it).

Leap 15.4 / SLE15SP4 also has it, though a slightly older version:  
https://build.opensuse.org/package/show/SUSE:SLE-15-SP4:GA/libvpl

So there is actually no need to add any extra repositories. For regular users, everything is pulled in automatically.

Maybe we can add a hint for software developers how to get the development headers, e.g. "zypper install libvpl-devel".

**Comment by cabelo** [comment]
> Maybe we can add a hint for software developers how to get the development headers

Perfect, great tip! It is generic for any opensuse user. Done and thanks again.

**Comment by cabelo** [comment]
Done! @StefanBruens

**Comment by cabelo** [comment]
@StefanBruens 
As a result of our conversation, I submitted the SLE-15-SP4:GA update as well. Thank you.

https://build.opensuse.org/request/show/1004403

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ You can install oneVPL:
 
 - from [oneVPL home page](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onevpl.html) as a part of Intel&reg; oneAPI Base Toolkit.
 - from source code.  See [Installation from Sources](INSTALL.md) for details.
+- from openSUSE distributiono the use the command: zypper install libvpl-devel
```
Comment (StefanBruens) [inline README.md:64]:
```suggestion
- from your Linux distribution:
  - SUSE/openSUSE: `zypper install libvpl-devel`
```

**Comment by cabelo** [comment]
Do I need to do anything else?

**Comment by jeffreymcallister** [comment]
Thanks for the discussion here, and for your help bringing VPL to the SLES/openSUSE communities.   Closing this thread.  Please feel free to start a new PR if there are any other changes you would like us to consider.

**Pull request closed without merging**